### ISSUE TITLE
Add site's brand mark

### ISSUE THREAD

**Issue by rafaelconde** [netlify/netlify-identity-widget#15]
![gotrue login example 2x](https://user-images.githubusercontent.com/2281080/30028904-e38a4332-917e-11e7-9c54-333e3ce68855.png)

We should add the option to allow developers to set a brand mark to personalize the widget.
Design wise I believe for the sake of scalability, we should make the available space, positioning, and shape non-customizable. 

Maybe in a separate effort allow to set a `tint-color` as well?

**Comment by skllpr** [comment]
I don't think that's completely true. Considering that the netlify logo/text and heart are rendered by this library, adding an option in this library shouldn't be outside of the scope of the library.

**Comment by gpalrepo** [comment]
Agreed. There should be an easier way to add our own logos. The inability to add own logos massively restricts the use of this service.
